Fluid.MvcSample/Startup.cs

```diff
@@ -12,29 +13,39 @@ public class Startup
         static Startup()
         {
             TemplateContext.GlobalMemberAccessStrategy.Register<Person>();
-            FluidViewTemplate.Factory.RegisterTag<MyTag>("mytag");
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
+            services.Configure<FluidViewEngineOptions>(x => x.Parser = p => 
+                p.RegisterEmptyBlock("mytag", async (s, w, e, c) =>
```

Comment:
maybe as static to promote best practices?

Reply:
Also I don't like the way I pass a lambda here (x.PArser = p =) I think I should just expose a parser instance so the only lambda is the Configure call